docs/tutorials/MenuGuide.md

```diff
@@ -0,0 +1,237 @@
+Creating a simple Menu
```

Comment:
Please add metadata. Check this page out: https://www.open.mp/docs/meta/Contributing#metadata